sources/counter.h

```diff
@@ -0,0 +1,106 @@
+
+#ifndef ODYSSEY_COUNTER_H
+#define ODYSSEY_COUNTER_H
+
+/*
+ * Odyssey.
+ *
+ * Scalable PostgreSQL connection pooler.
+ */
+
+#include <malloc.h>
+#include <kiwi.h>
+#include <zconf.h>
+#include "macro.h"
+
+typedef struct od_counter od_counter_t;
+typedef size_t od_hash_table_key_t;
+
+#define OD_DEFAULT_HASH_TABLE_SIZE 15
```

Comment:
I think this must be near Hashtable , right?

Reply:
yeah, right